app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,27 @@
+class MessagesController < ApplicationController
+  def new
+    @message = Message.new
+  end
+
+  def create
+    @message = Message.new(message_params)
+
+    if @message.save
+      flash[:notice] = "Message created successfully"
+      redirect_to messages_path
+    else
+      flash.now[:error] = "Could not create message"
```

Comment:
Sure. The only validation is presence, and users won't be able to submit the form with blank fields, but I will add the error messages too just in case.